src/parser/scanner_support.c

```diff
@@ -133,8 +198,17 @@ struct st_hint_msg
     check_buffer (snprintf (NULL, 0, fmt, str1, str2));
     m_used += sprintf (msg_ptr + m_used, fmt, str1, str2);
   }
+  void addendstring ()
+  {
+    check_buffer (2);
```

Comment:
what does mean 2?

Reply:
Check the space required to append a string to msg_ptr. Make room for '\0'.